### DISCUSSION TITLE
small question about what toonsquid runs on

### DISCUSSION THREAD

**Discussion by TpoapAnimates** [keiwando/toonsquid#700] [Q&A] [answered]
will Toonsquid work on a 6th generation Apple Ipad thats updated to the newest version?
I don't have an iPad so that's why I'm checking so I can buy the right one

**Comment by sichtbeton** [comment]
I use it on ipad Air 3 without any glitches yet.

**Reply by TpoapAnimates** [reply]
alr that's great

**Comment by Cat-Butler** [comment, marked as answer]
According to the [App Store](https://apps.apple.com/app/id1573778812), the current version of ToonSquid runs on iPads using iPadOS 15 or above, which includes 6th generation iPads.
For the longer term, Apple has stopped iPadOS updates at 17 for 6th generation iPads, they are not supported for iPadOS 18.
They should still get security updates for at least a couple of years.

**Reply by TpoapAnimates** [reply]
thank you so much
